### PR TITLE
[OSDOCS-7881]: Unify `ccoctl` install step order across providers

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.14+

Issue:
[OSDOCS-7881](https://issues.redhat.com//browse/OSDOCS-7881)

Link to docs preview:
"Alternatives to storing administrator-level secrets in the kube-system project" in:
* [Installing a cluster on AWS with customizations](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations#installing-aws-manual-modes_installing-aws-customizations)
* [Installing a cluster on AWS with network customizations](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-network-customizations#installing-aws-manual-modes_installing-aws-network-customizations)
* [Installing a cluster on AWS in a restricted network](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-restricted-networks-aws-installer-provisioned#installing-aws-manual-modes_installing-restricted-networks-aws-installer-provisioned)
* [Installing a cluster on AWS into an existing VPC](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-vpc#installing-aws-manual-modes_installing-aws-vpc)
* [Installing a private cluster on AWS](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-private#installing-aws-manual-modes_installing-aws-private)
* [Installing a cluster on AWS into a government region](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-government-region#installing-aws-manual-modes_installing-aws-government-region)
* [Installing a cluster on AWS into a Secret or Top Secret Region](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-secret-region#installing-aws-manual-modes_installing-aws-secret-region)
* [Installing a cluster on AWS China](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-china#installing-aws-manual-modes_installing-aws-china-region)
* [Installing a cluster on AWS with remote workers on AWS Outposts](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-outposts-remote-workers#installing-aws-manual-modes_installing-aws-outposts-remote-workers)
* [Installing a cluster on GCP with customizations](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-customizations#installing-gcp-manual-modes_installing-gcp-customizations)
* [Installing a cluster on GCP with network customizations](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-network-customizations#installing-gcp-manual-modes_installing-gcp-network-customizations)
* [Installing a cluster on GCP in a restricted network](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-restricted-networks-gcp-installer-provisioned#installing-gcp-manual-modes_installing-restricted-networks-gcp-installer-provisioned)
* [Installing a cluster on GCP into an existing VPC](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-vpc#installing-gcp-manual-modes_installing-gcp-vpc)
* [Installing a cluster on GCP into a shared VPC](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-shared-vpc#installing-gcp-manual-modes_installing-gcp-shared-vpc)
* [Installing a private cluster on GCP](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-private#installing-gcp-manual-modes_installing-gcp-private)

Added Azure root secret value to [Clusters that use short-term credentials: Verifying the credentials configuration](https://65120--docspreview.netlify.app/openshift-enterprise/latest/installing/validating-an-installation#cco-ccoctl-install-verifying_validating-an-installation), reformatted for additional context.

QE review:
- [x] QE has approved this change.

Additional information:
- Planned for 4.14.z, add 4.15 label post GA

### CODE REVIEW

File: installing/installing_aws/installing-aws-secret-region.adoc
Comment (jeana-redhat) [inline installing/installing_aws/installing-aws-secret-region.adoc:1]: GH calculated this diff oddly, but as far as I can tell the changes are the same

Comment (jeana-redhat) [reply installing/installing_aws/installing-aws-secret-region.adoc:1]: Weird in a different way after rebase, need to check on this

File: modules/cco-ccoctl-creating-individually.adoc
Comment (jeana-redhat) [inline modules/cco-ccoctl-creating-individually.adoc:1]: This should stay

File: installing/installing_aws/installing-aws-secret-region.adoc
Comment (jeana-redhat) [reply installing/installing_aws/installing-aws-secret-region.adoc:1]: Checked line by line, this is correct afaik but will compare against 4.14 render when preview is rebuilt

**Comment by jstuever** [comment]
lgtm

**Comment by jeana-redhat** [comment]
@jianping-shu this is ready for verification, PTAL when you have some time. Our repo freeze is in a couple of hours, so the earliest this can merge is the second half of next week.

**Comment by jeana-redhat** [comment]
refreshed rebase

**Comment by jianping-shu** [comment]
LGTM

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.15` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.14. And any PR going into `main` must also target the latest version branch (enterprise-4.15).

If the update in your PR does NOT apply to version 4.15 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main

File: installing/installing_aws/installing-aws-network-customizations.adoc [hunk 1]
```diff
@@ -91,21 +58,49 @@ include::modules/installation-aws-config-yaml.adoc[leveloffset=+2]
...
+* To manage long-term cloud credentials manually, follow the procedure in xref:../../installing/installing_aws/installing-aws-network-customizations.adoc#manually-create-iam_installing-aws-network-customizations[Manually creating long-term credentials].
 
-* If you will manage cloud credentials manually, follow the procedure in xref:../../installing/installing_aws/installing-aws-network-customizations.adoc#manually-create-iam_installing-aws-network-customizations[Manually creating long-term credentials].
+* To implement short-term credentials that are managed outside the cluster for individual components, follow the procedures in  xref:../../installing/installing_aws/installing-aws-network-customizations.adoc#installing-aws-with-short-term-creds_installing-aws-network-customizations[Configuring an AWS cluster to use short-term credentials].
```
Comment (michaelryanpeter) [inline installing/installing_aws/installing-aws-network-customizations.adoc:71]: looks like you have an extra space before the start of the `xref`.

**Review by michaelryanpeter — COMMENTED** [review]
LGTM

/label peer-review-done
/remove-label peer-review-needed
/remove-label peer-review-in-progress

File: installing/installing_aws/installing-aws-network-customizations.adoc (hunk 1 above)
Comment (jeana-redhat) [reply installing/installing_aws/installing-aws-network-customizations.adoc:71]: :detective: :joy: :+1:

**Comment by jeana-redhat** [comment]
refresh for merge

**Pull request merged**